experiment/simulation/chemdoodleWeb-9.2.0/ChemDoodleWeb.css

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Why are we copying the chemdoodleweb components instead of linking them?

Reply:
I didn't found any CDN link for chemdoodle webcomponent. In their documentation, they suggested to download their files and use it locally. Hence I did the same. If required, we can host is separately and use a custom CDN.

Reply:
Please do host them at a common place if inside Virtual Labs. Deploying them on CDN may be bad.
We also need to see if we follow their licensing criteria.